src/NUnitTestExtractor/NUnitTestExtractorTests/TestExtractorTests.cs

```diff
@@ -31,6 +32,32 @@ private static void InvokeGetTestsUsingFileOutput(StringBuilder builder, List<st
         }
 
         /// <summary>
+        /// Determines which regex string return depending on the Level 
+        /// </summary>
+        /// <param name="myLevel">The level which to use to determine which regex to return</param>
+        /// <returns>The proper regex string format which is then used to compare strings to</returns>
+        private static Regex VerifyOutputFormat(NUnitTestExtractorApp.Level myLevel)
+        {
+            Regex regex = null;
+
+            switch (myLevel)
+            {
+                case NUnitTestExtractorApp.Level.Namespace:
+                    regex = new Regex(@"^(\w+)[\w+\s\|\s\w+$]");
```

Comment:
After putting quotes around the DLL name, this regex can change to:  `^"(.+)"\s+\|\s+(\w+)$`
Similar changes to other regexes below.